crypto/currency-exchange/README.md

```diff
@@ -0,0 +1,137 @@
+# Currency Exchange
+
+## Firestore Collections
+
+- Currency
+  > this will contain details about the accepted currencies at crypto-website
+
+## Currency Object
+
+```
+{
+    "id": <currency_id>,
+    "name": <currency_name>,
+    "quantity": <available_quantity>
+}
+```
+
+## Currency Exchange Object
+
+```
+{
+    from: {
+        "id": <currency_id>,
+        "name": <currency_name>
```

Comment:
Why do we need to repeat the name here, can we not extract this from the currency id?